dependencies.sh

```diff
@@ -21,5 +21,5 @@ export NODE_VERSION_PRECISE=14.16.1
 echo "NODE_VERSION_PRECISE=$NODE_VERSION_PRECISE" >> "$GITHUB_ENV"
 
 # Python version for mapmerge and other tools
-export PYTHON_VERSION=3.9
```

Comment:
Because it's not actual versioning like "give me any package that is 3.9 or above, but below 4.0".

It literally tries to download a python zip via ftp that is named "3.9.0". There is no zip named "3.9" so fails on that. Maybe a zip named "3.9" existed before in the past, no idea.

This is how it is on tg so if it works for them, it'll for work for us too.